api/src/main/java/com/thesurvey/api/service/converter/CertificationTypeConverter.java

```diff
@@ -1,39 +0,0 @@
-package com.thesurvey.api.service.converter;
-
-import com.thesurvey.api.domain.EnumTypeEntity.CertificationType;
-import java.util.List;
-import java.util.stream.Collectors;
-import org.springframework.context.annotation.Configuration;
-
-@Configuration
-public class CertificationTypeConverter {
```

Comment:
Why did you remove this file?

Reply:
Since the converter only gets involved when retrieving data from the database and does not affect the storage of CertificationType in the database.. so I removed it

Reply:
Then, wouldn't this be much closer to `TypeUtil` class?

Reply:
I will keep the file because it seems ambiguous whether this class is more suitable for `TypeUtil` or `Converter`..